packages/typescript-resolver-files/src/generateResolverFiles/postProcessFiles.ts

```diff
@@ -117,12 +109,9 @@ const ensureExportedResolver = (
 
   if (!variableStatement) {
     // Did not find variable statement with expected identifier, add it to the end with a warning
-    const addedVariableStatement = sourceFile.addStatements(
-      resolverFile.meta.variableStatement
-    );
+    sourceFile.addStatements(resolverFile.meta.variableStatement);
 
     return {
-      variableStatement: addedVariableStatement as unknown as VariableStatement, // We know it's a variable statement because we just added it.
```

Comment:
Cannot make this assumption here because `resolver.meta.variableStatement` on line https://github.com/eddeee888/graphql-code-generator-plugins/pull/297/files#diff-c049d5c9a7fad48dce1d21c27d36a732ec940276889fca382af509816af036e7L120-L122 is just a string and could contain non-variable statement e.g. comments